src/configurations/crc-researcher/routeButtonControlWrapperProps.ts

```diff
@@ -0,0 +1,16 @@
+import { SynapseConfig } from 'types/portal-config'
+import { RouteButtonControlWrapperProps } from 'portal-components/RouteButtonControlWrapper'
+const routeButtonControlProps: RouteButtonControlWrapperProps = {
+  // this has to get overriden,
+  synapseConfig: {} as SynapseConfig,
+  colors: ['#47337D', '#407BA0', '#7798AC', '#77BBBF', '#58A158', '#3C4A63'],
+  customRoutes: [],
```

Comment:
Shouldn't customRoutes have names for each route to render the buttons on the /Participants page? 

It should be
['Uncategorized', 'Selected', 'Invited','Hidden']

Otherwise you'll only navigate through the tabs at the top